lua/rocks/operations.lua

```diff
@@ -305,6 +308,8 @@ operations.sync = function(user_rocks)
                 percentage = get_progress_percentage(),
                 message = is_downgrading and ("Downgraded: %s"):format(key) or ("Upgraded: %s"):format(key),
             })
+
+            vim.opt.runtimepath:append(vim.fs.joinpath(config.rocks_path, "lib", "luarocks", "rocks-5.1", "*", user_rocks[key].name:lower()))
```

Comment:
issue: I think we only want to do this for newly added plugins.

Reply:
I just realized this bit is  extraneous as it's done in the install function anyway